jina/resources/logging.default.yml

```diff
@@ -1,7 +1,7 @@
 handlers:  # enabled handlers, order does not matter
   - StreamHandler
   - SysLogHandler
-  - FluentHandler
+#  - FluentHandler TODO(Han): I temporally disable it for suppress the warning and better ux for newbies
```

Comment:
`jinad` is now bundled with fluentd to privide the `log streaming` endpoint. how will this handler be enabled?

Reply:
by setting different environment variable `JINA_LOG_CONFIG`
![image](https://user-images.githubusercontent.com/2041322/99903624-bca45d80-2cc5-11eb-901c-ce728cf16863.png)

Reply:
i revert the change and suppress the warning for this particular import now